src/common/common-util/src/main/java/ee/ria/xroad/common/conf/globalconf/ConfigurationFile.java

```diff
@@ -136,8 +136,8 @@ private static ContentIdentifier getContentIdentifier(String value) {
 
         Map<String, String> p = new HashMap<>();
 
-        String id = HttpField.valueParameters(value, p);
-        String instance = p.get(PARAM_INSTANCE);
+        String id = HttpField.getValueParameters(value, p);
+        String instance = StringUtils.unwrap(p.get(PARAM_INSTANCE), '\'');
```

Comment:
how the params have changed that now we need extra unwraping?

Reply:
Not params, the utility for extracting them changed so it leaves for some reason those quotes